Sources/TokamakDemo/Counter.swift

```diff
@@ -29,6 +29,8 @@ public struct Counter: View {
           Button("Increment") { count += 1 }
           Text("\(count)")
         }
+        .onAppear { print("Counter.VStack onAppear") }
+        .onDisappear { print("Counter.VStack onDisappear") }
```

Comment:
Where do `print` statements get outputted?

Reply:
In Web Inspector console, all `print` output is redirected with `console.log`.